gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -1810,6 +1814,9 @@ gst_tensor_filter_start (GstBaseTransform * trans)
     GST_OBJECT_UNLOCK (self);
   }
 
+  /* Property conditions are required for setting */
+  gst_tensor_filter_set_async_output_callback_notify_util (priv, self, nnstreamer_filter_async_output_callback);
+
```

Comment:
A callback is supposed to be configured by subplugins.
Why the framework is configuring callback function... statically?
(if statically allocated callback function is ok, you don't need a callback function)

Reply:
Create a callback function in tensor_filter and pass the callback function pointer to the sub plugin using eventHandler.
When the sub plugin has multiple outputs for the input, it passes the output to tensor_filter as a callback whenever an output is generated.
At this time, the memory for the output is allocated in the sub-plugin.
```
 input ---> tensor_filter     (callback: gst_pad_push(gstbuffer)
                  |                               ^
                  |                                |
                  --> sub plugin --> output
 ```